README.md

```diff
@@ -1,2 +1,37 @@
 # hw01
-Repository for homework 01
+Repository for homework 01  
+
+# Who is Yeon Jang?  
+
+Hi all, I'm Yeon. I am a MAPSS student with a concentration in Psychology. For my thesis, I'm currently studying the effects of community violence exposure on the neurodevelopment of adolescents.  
+
+Prior to UChicago, I completed my undergraduate education in Minnesota at Carleton College. In case you've never heard of Carleton, here's a link to their website: [Carleton College Website](https://www.carleton.edu/). 
+
+After UChicago, I'm not quite sure yet what I'll end up doing, but at this point, it's likely that I apply to doctorate programs.   
+
+Outside of school stuff, I enjoy eating, running, listening to music, and watching tv. *I don't have a lot of hobbies...*  
+
+Here's a few of my favorite things:  
+
+* **TV shows**
+    + Friends
```

Comment:
You don't need to change `*` to `+`. You can simply write 
```
* **TV shows**
   *Friends
   *Community
```
And it will still work.